GitUI/AutoCompletion/CommitAutoCompleteProvider.cs

```diff
@@ -81,7 +81,7 @@ private static IEnumerable<string> ReadOrInitializeAutoCompleteRegexes()
                 return File.ReadLines(path);
             }
 
-            Stream s = Assembly.GetEntryAssembly().GetManifestResourceStream("GitExtensions.AutoCompleteRegexes.txt");
+            Stream s = Assembly.GetEntryAssembly()?.GetManifestResourceStream("GitExtensions.AutoCompleteRegexes.txt");
             if (s == null)
             {
                 throw new NotImplementedException("Please add AutoCompleteRegexes.txt file into .csproj");
```

Comment:
We avoid a `NullReferenceException`, but then we immediately throw `NotImplementedException`. Was that intended?

Reply:
Yes, it is intended because it gives a more clear information what failed.
As a positive side effect, the debugger does not stop on `NotImplementedException` by default when debugging tests.
Though it is only a workaround.